tripy/tripy/frontend/module/sequential.py

```diff
@@ -0,0 +1,169 @@
+from typing import Union, Tuple, Dict, Iterator, Any
+from dataclasses import dataclass
+
+from tripy import export
+from tripy.frontend.module.parameter import Parameter
+from tripy.frontend.module import Module
+
+
+@export.public_api(document_under="modules/sequential.rst")
+@dataclass
+class Sequential(Module):
+    r"""
+    A module to stack multiple layers or modules in a sequential order. The `Sequential`
+    container can accept either a list of modules or a dictionary of named modules. Modules are
+    added in the order they are passed, and each is called sequentially during the forward pass.
+    """
+
+    modules: Union[Module, Dict[str, Module]]
+    r"""The modules to include in the sequence"""
```

Comment:
Should this be publicly accessible/documented? If not, we can drop these two lines.

Reply:
dropping, should not be accesible